tests/acl/test_acl.py

```diff
@@ -46,7 +47,7 @@
 
 # TODO: We really shouldn't have two separate templates for v4 and v6, need to combine them somehow
 ACL_RULES_FULL_TEMPLATE = {
-    "ipv4": "acltb_test_rules.j2",
+    "ipv4": "acltb_test_rules_permit_loopback.j2",
```

Comment:
Why not add the loopback ACL rule to the original file?

Reply:
Hi @bingwang-ms , the acltb_test_rules.j2 has shared by tests/ssh/test_ssh_stress.py too, if keep the change in the original file, it would affect tests/ssh/test_ssh_stress.py.